src/components/Content/Content.js

```diff
@@ -10,20 +10,31 @@ export default function Content() {
   const [todos, setTodos] = useState([]);
 
   const handleCreateTodo = (description) => {
-    const newTask = {
+    const newTodo = {
       description,
       id: new Date().getTime(),
       completed: false,
     };
 
-    setTodos([...todos, newTask]);
+    setTodos([...todos, newTodo]);
   };
 
   const handleRemoveTodo = (id) => {
     const filteredTodos = todos.filter((todo) => todo.id !== id);
     setTodos(filteredTodos);
   };
 
+  const handleEditTodo = (id, description) => {
+    const editedTodos = todos.map((todo) => {
+      if (id === todo.id) {
+        return { ...todo, description };
+      }
+      return todo;
+    });
+    console.log(description);
```

Comment:
We should avoid `console.log` in PR, they are for local development/testing purpose only